lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -174,7 +184,9 @@ export class RedirectClient extends StandardInteractionClient {
                 navigationClient: this.navigationClient,
                 redirectTimeout: this.config.system.redirectNavigationTimeout,
                 redirectStartPage: redirectStartPage,
-                onRedirectNavigate: request.onRedirectNavigate,
+                onRedirectNavigate:
+                    this.config.auth.onRedirectNavigate ||
+                    request.onRedirectNavigate,
```

Comment:
This should probably be the other way around for now as the pattern everywhere else is that request parameters override PCA config parameters

```suggestion
                    request.onRedirectNavigate ||
                    this.config.auth.onRedirectNavigate,
```